rel-eng/custom/custom.py

```diff
@@ -88,7 +88,7 @@ def filter_archives(path):
         rel_date = datetime.utcnow().strftime("%Y%m%d%H%M")
         self.release = rel_date + gitrev
         print("Building release: %s" % self.release)
-        run_command("sed -i '1i %%global nightly .%s/' %s" % (self.release, self.spec_file))
+        run_command("sed -i '1i %%global nightly .%s' %s" % (self.release, self.spec_file))
```

Comment:
What is this change?

Reply:
Forgotten rebase: 67fa5fdc2b1c4ecfe6da3f4d781dc5f3655ee55e was already merged.